src/content-scripts/ActionMgr.js

```diff
@@ -112,12 +112,12 @@ class ActionMgr {
 					new Set(
 						SelectedElements
 							.Links
-							.map(elem => elem.getAttribute('href'))
+							.map(elem => elem.href)
 					)
 				);
 
-				if(e.ctrlKey)
-					this.CopyToClipboard(links);
+				if ((e.ctrlKey && Prefs.DefaultAction == OPEN_LINKS) || (!(e.ctrlKey) && Prefs.DefaultAction == COPY_LINKS))
+				    this.CopyToClipboard(links);
```

Comment:
These checks are failing because eslint doesn't know about these globals.  If you edit the eslint settings you'll see a globals declaration section which can have those added to as globals then this should pass.